app/models/manifest.rb

```diff
@@ -1,7 +1,9 @@
 require 'date'
 
 class Manifest < ActiveRecord::Base
-  validates :tracking_number, presence: true
+  include Searchable
+
+  validate :tracking_number, :validate_tracking_number_unique
```

Comment:
how is `validate_tracking_number_unique` giving a different result than doing `validates :tracking_number, presence: true, uniqueness: true` ?